src/binaryen-c.cpp

```diff
@@ -5670,6 +5678,14 @@ BinaryenIndex BinaryenModuleAddDebugInfoFileName(BinaryenModuleRef module,
   return index;
 }
 
+const char* BinaryenModuleGetDebugInfoSymbolName(BinaryenModuleRef module,
+                                                 BinaryenIndex index) {
+  const auto& debugInfoSymbolNames = ((Module*)module)->debugInfoSymbolNames;
+  return index < debugInfoSymbolNames.size()
+           ? debugInfoSymbolNames.at(index).c_str()
+           : nullptr;
+}
```

Comment:
Are these changes used/tested? I don't see them in `binaryen-c.h`, so I'm not sure how they are meant to be used. I also don't see a test.

Reply:
-1 as uint32 would probably be ok, as that is likely not a valid value? (at least in DWARF it isn't in various contexts iirc).

But, if we don't have a need for the C API atm, I would just remove this. We can always look into it later if it come sup.

Reply:
I don't need the C API, I wanted to implement it for completeness.

I reverted the C API changes now. PTAL.